lms/djangoapps/courseware/tests/test_tabs.py

```diff
@@ -484,8 +484,6 @@ def setUp(self):
             [{'type': CoursewareTab.type}],
             # missing course_info
             [{'type': CoursewareTab.type}, {'type': 'discussion', 'name': 'fake_name'}],
-            # incorrect order
-            [{'type': CourseInfoTab.type, 'name': 'fake_name'}, {'type': CoursewareTab.type}],
```

Comment:
Could this list be reversed in order to keep the test of tabs in the wrong order? I'd rather not see test coverage go down with this PR.